### DISCUSSION TITLE
Discuss support of GFA format in Icarus output

### DISCUSSION THREAD

**Discussion by alexeigurevich** [ablab/quast#169] [General]
Graphical Fragment Assembly (GFA) format describes sequence overlap graphs (assembly graphs). Specification and examples are here: https://github.com/pmelsted/GFA-spec

GFA is supported by ABySS 1.9.0, Bandage and many other tools. So, implementing its support in QUAST (for Icarus' contig alignment viewer) sounds reasonable and useful for the community.

This issue is created for a further discussion and any suggestions about GFA data representation in Icarus.

This enhancement idea was suggested by @sjackman (Shaun Jackman)

**Comment by alexeigurevich** [comment]
The initial idea is to show contigs which are connected to each other according to GFA file. These connections may be shown pretty much the way we do for presenting misassembled blocks related to the same contig: special section in Contig info window and grey arrows on Alignment overview pane.

**Comment by sjackman** [comment]
Read the genome assembly contig overlap graph in Graphical Fragment Assembly (GFA) format and display the mean read coverage of the contig and links to the adjacent overlapping contigs. GFA is an emerging standard for genome assembly graphs originally proposed by Heng Li and now maintained by a community including myself. It is implemented by a number of assemblers and tools, including the visualization tool Bandage, which I also use for assessing genome assembly correctness. Icarus and Bandage complement each other nicely. See https://github.com/pmelsted/GFA-spec
